_posts/2020-09-18-ai-ethics-optimization-problem/ai-ethics-optimization-problem.Rmd

```diff
@@ -0,0 +1,172 @@
+
+Phillip Rogaway, who I'll mention again in the concluding section, talks a lot about how technology optimism dominates among his students [@Rogaway]; he seems to be as intrigued by it as I am. Personally, I think that whether someone "intrinsically" tends to be a technology optimist or a pessimist is a persistent trait; it seems to be a matter of personality and socialization (or just call it *fate*: I don't want to go into any nature-nurture debates here). That glass, is it half full or half empty?
+
+All I could say to a hardcore technology optimist is that their utopia may be another person's dystopia. Especially if that other person is poor, or black, or poor and a black woman ... and so on.
+
+Let me just end this section with a citation from a [blog post](https://ali-alkhatib.com/blog/anthropological-intelligence) by Ali Alkhatib centered around the launch of the [Stanford institute for human-centered artificial intelligence (HAI)](https://hai.stanford.edu/). Referring to the [director's accompanying blog post](https://hai.stanford.edu/blog/smart-interfaces-human-centered-ai), he writes
+
+> James opens with a story of an office that senses you slouching, registers that you're fatigued, intuits that your mood has shifted, and alters the ambiance accordingly to keep you alert throughout the day.
+
+You can head to Alkhatib's post (very worth reading) and read the continuation, expanding on the scenario. But for some people, this single sentence may already be enough.
+
+## And now?
+
+At some point, an author is expected to wrap up and present ideas for improvement. With most of the topics touched upon here, this is yet another intimidating task. I'll give it a try anyway. The best synthesis I can come up with at the moment looks about like this.
+
+First, some approaches filed by Green under "formalist" can still make for a good start, or rather, can constitute a set of default measures, to be taken routinely. Most prominently, these include dataset [@2018arXiv180309010G] and model [@2018arXiv181003993M] documentation.
+
+Beyond the technical, I like the advice given in Baumer and Silberman's *When the Implication is Not to Design* [@Baumer]. If the consequences, especially on socially marginalized groups, of a technological approach are unforeseeable, think whether the problem can be solved in a "low-tech" way. By all means, do *not* start with the solution and then, go find a problem.
+
+In some cases, even that may not be enough. With some goals, don't look for alternative ways to achieve them. Sometimes the goal itself has to be questioned.
+
+The same can be said for the other direction. With some technologies, there is no goal that could justify their application. This is because such a technology is *certain* to get misused. Facial recognition is one example.
```

Comment:
I agree to disagree, but no need to change :) I think is a bit too far fetched to not use facial recognition for anything at all. The analogy here is nuclear power, we don't want to use it for nuclear warheads, but I'm OK with nuclear power plants. Although, many might disagree with my point of view on facial recognition.

Reply:
Yes, agree to disagree here too :-)

To me the point is that for some technologies, no goal can justify their development because their (immediate) misuse is certain from the beginning.